### DISCUSSION TITLE
这个壁纸软件就是一个挖矿软件，在后台占用我cpu100%

### DISCUSSION THREAD

**Discussion by 2226075815** [rocksdanister/lively#2457] [General]
在后台占用我cpu100%，怪不得我突然发现我电脑风扇一直疯狂转动，我明明没有运行其它东西

**Comment by rocksdanister** [comment]
dup: https://github.com/rocksdanister/lively/wiki/Performance

The software is opensource, anyone is free to audit what it is doing.
